src/app/admin/index.ts

```diff
@@ -0,0 +1,17 @@
+import { checkedToUsername } from "@domain/users"
+import { AccountsRepository } from "@services/mongoose"
+
+export const getUserByUsername = async function (username: string) {
+  const usernameValid = checkedToUsername(username)
+  if (usernameValid instanceof Error) throw usernameValid
+
+  const username_ = username as Username
+
+  const accounts = AccountsRepository()
```

Comment:
agree for the phone. but we may not want to have user have username (only phone or email) to avoid the confusion on the use of username: it's to receive a payment, not to login.